src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -59,4 +62,8 @@ public static String getEditPersonDescriptorDetails(EditPersonDescriptor descrip
         }
         return sb.toString();
     }
+    public static String getRemoveTagCommand(Index index, String tag) {
+        return RemoveTagCommand.COMMAND_WORD + " " + index.getOneBased()
+            + " " + (String.join(" ", "t/" + tag.toString()));
+    }
```

Comment:
Prehaps having a Javadoc here would be great to show what this method is for/doing.